src/button/icon-button.component.ts

```diff
@@ -27,7 +27,6 @@ import { ButtonSize, ButtonType } from "./button.types";
 	<cds-tooltip
 		class="cds--icon-tooltip"
 		[description]="description"
-		[disabled]="disabled"
```

Comment:
I think we should continue to keep this, this was intentional. So we should create a an input 
```ts
@Input() showTooltipWhenDisabled = false
```


```html
[disabled]="showTooltipWhenDisabled ? true : disabled"
```

Reply:
The logic in your example was off btw, `showTooltipWhenDisabled` should be `false` in the ternary :D